packages/flutter_tools/lib/src/base/process.dart

```diff
@@ -449,7 +451,12 @@ class _DefaultProcessUtils implements ProcessUtils {
     }
 
     if (failedExitCode && throwOnError) {
-      runResult.throwException('The command failed');
+      String message = 'The command failed';
```

Comment:
Did you mean to check `verboseExceptions` somewhere?

Reply:
You're right, the `throwOnError` check here is redundant, since we already know it's true from the check on 453. This should be `if (verboseExceptions) {`. Added another unit test.